### PR TITLE
🔖 Publish 2.4.3

### PR DESCRIPTION
**Pull request by isaporiti**
This is to address these comments: https://linear.app/superlayer/issue/SUP-1350/new-ui-guidelines-colours-redesign#comment-e0414321

### CODE REVIEW

File: src/styles/bds.sass [hunk 1]
```diff
@@ -23,4 +23,10 @@
...
 // Import Bulma extension
-@import "@creativebulma/bulma-tooltip/src/sass/index.sass"
\ No newline at end of file
+@import "@creativebulma/bulma-tooltip/src/sass/index.sass"
+
+body
+    letter-spacing: 0
+
+input, textarea, button, .button
+    border-radius: 4px !important
```
Comment (isaporiti) [inline src/styles/bds.sass]: What do you think of this @giulio-sl ? I think it could go somewhere else, but I'm not sure where.

Comment (giulio-sl) [reply src/styles/bds.sass]: Good question! I need to investigate this 😅

Comment (isaporiti) [reply src/styles/bds.sass]: I'll have a look at alternatives for this now

Comment (isaporiti) [reply src/styles/bds.sass]: @giulio-sl I ended up moving the global styles to its own file: https://github.com/superlayerhq/biings-ds/pull/13/commits/a9d79549a94c8b03159e97702654dd2d595697e3

**Review by giulio-sl — APPROVED** [review]

**Pull request merged**